.travis.yml

```diff
@@ -1,47 +1,29 @@
-language: generic
-
-os:
-  - linux
-  - osx
```

Comment:
What's your plan for OSX ?

Reply:
reuse the conda version that was used before with a travis matrix...

Reply:
@videlec, any progress on OSX?

Reply:
@vinklein was supposed to work on it last week.

Reply:
Fixed on PR #167